plugins/kafka/kafka.ecllib

```diff
@@ -212,9 +212,10 @@ EXPORT Consumer(VARSTRING topic, VARSTRING brokers = 'localhost', VARSTRING cons
         END;
 
         // Set the offset for each partition on each node
-        result := PROJECT
+        result := NORMALIZE
             (
-                distOffsets,
+                NOFOLD(distOffsets),
```

Comment:
It appears that only wrapping distOffsets in NOFOLD within the local PROJECT will accomplish what I need.  Now I'm wondering what I did wrong in my earlier tests that caused this to not work correctly, but that's a topic for another conversation.  I'll update the commit and rebase shortly.